3rdparty/python/requirements.txt

```diff
@@ -8,7 +8,7 @@ mox==0.5.3
 
 # Until https://github.com/pantsbuild/pex/issues/28 is addressed, we cannot
 # do pex<1.1.0.
-pex>=1.0.2,<1.0.999999
+pex>=1.0.3,<1.0.999999
```

Comment:
out of curiosity, why do we need the range here? would it not be cleanest to just pin pex==1.0.3?